whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -72,12 +84,15 @@ public FilterAuthFunction() {
     @Override @Nonnull
     public RpslObject apply(final RpslObject rpslObject) {
         final List<RpslAttribute> authAttributes = rpslObject.findAttributes(AttributeType.AUTH);
-        if (authAttributes.isEmpty()) {
+        if (authAttributes.isEmpty() && override == null) {
```

Comment:
Why `override == null` ? There isn't a similar check when using password auth?

Reply:
for the early return that we are doing there is necessary to check `override` as well.
If not we could be early returning in case `override` is there.
Override doesn't have nothing to do with authAttributes.

In the update we are checking that we are using override yes, `update.isOverride()` 
If this is true then we do an overrideAuthentication

Reply:
It's a confusing change. Does authAttributes have to be empty *AND* override be null ? Can you add a separate `if` statement just for override ?

Reply:
yes, authAttributes have to be empty AND override be null for the early return.
another way to read it...we need override OR at least one auth Attribute to make the authentication

Reply:
Discussed with Mahesh, I changed this